samples/net/zperf/src/main.c

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief Zperf sample.
+ */
+
+#if defined(CONFIG_USB)
```

Comment:
Do we really need to guard this two includes?

Reply:
No. I've removed all conditional includes.